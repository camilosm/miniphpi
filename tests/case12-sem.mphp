$a = array(0 => "zero");
$b = array(1 => "um");
$a[$b] = 2;