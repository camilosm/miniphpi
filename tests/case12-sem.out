03: Operação inválida